src/types.rs

```diff
@@ -10,6 +10,32 @@ use std::{
 
 use crate::execution::RuntimeModule;
 
+#[cfg(feature = "execution-trace")]
+#[derive(Debug, Clone, PartialEq)]
+pub enum AbiTraceType {
+    None,
+    Bool(bool),
+    U8(u8),
+    I32(i32),
+    I64(i64),
+    U64(u64),
+    F64(f64),
+    ByteArray(Vec<u8>),
+    ByteArrays(Vec<Vec<u8>>),
```

Comment:
ByteArrayArray?

Reply:
Type names have already been defined with the explorer guys ...

Reply:
ok, they're not consistent
let's live with that